doc/admin-guide/files/storage.config.en.rst

```diff
@@ -184,3 +184,9 @@ following rules are stored in :manpage:`devfs.conf(5)`::
 
    # Assign /dev/ada1 and /dev/ada2 to the tserver user
    own    ada[12]  tserver:tserver
+
+Advanced
+--------
+
+When ``runroot`` generated from ``traffic_layout`` is used, :file:`storage.config` should be updated
```

Comment:
"updated accordingly pointing"?

Reply:
Yeah, I need to polish the grammar a bit.